robot/marker_setup/teams.py

```diff
@@ -1,17 +1,19 @@
 import enum
 
-# spell-checker:words ZHORA,PRIS,BATEY
+"""
```

Comment:
Good explanation. This code base mostly follows the style set out by PEP8 the official style guide for Python to make it easier to read. It specifies that comments on functions in multiline comments (doc strings) should go below the function header: https://peps.python.org/pep-0008/